gent_base.theme

```diff
@@ -1021,8 +1021,9 @@ function gent_base_preprocess_vesta(&$variables) {
   /* @var $entity Drupal\dg_vesta\Entity\Vesta */
   $entity = $variables['elements']['#vesta'];
 
+  $locale = \Drupal::languageManager()->getCurrentLanguage()->getId();
```

Comment:
```suggestion
  // Because vesta entities are language undefined we need to prefix the language code.
  $locale = \Drupal::languageManager()->getCurrentLanguage()->getId();
```